### PR TITLE
Welder Satchel

### PR DESCRIPTION
**Pull request by Tunguso4ka**
## About the PR

Ported welder satchel from the cm-ss13.
Idk (or dont remember) why TODO on welderpack says that welders are broken, so i leaved it here too.

## Media

зображення

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added welder satchel for Combat Technicians.

### CODE REVIEW

File: Resources/Prototypes/_RMC14/Entities/Clothing/Back/satchels.yml [hunk 1]
```diff
@@ -117,4 +117,39 @@
     whitelist:
       tags:
       - Grenade
-  - type: FixedItemSizeStorage
\ No newline at end of file
+  - type: FixedItemSizeStorage
+
+- type: entity
+  parent: CMSatchel
+  id: CMSathelWelder
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Clothing/Back/satchels.yml:124]:
```suggestion
  id: RMCSathelWelder
```

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Squad/engineer.yml [hunk 2]
```diff
@@ -195,7 +195,7 @@
       - id: CMBackpackMarineTech
       - id: CMSatchelMarineTech
       - id: CMBackpackWelder
-    #- id: CMTechnicianWelderSatchel
+      - id: CMSatchelWelder
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Squad/engineer.yml:198]:
```suggestion
      - id: RMCSatchelWelder
```

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Squad/prep.yml [hunk 3]
```diff
@@ -160,8 +160,8 @@
       entries:
       - id: CMBackpackWelder
         amount: 1
-#      - id: Technician Welder-Satchel
-#        amount: 2
+      - id: Technician Welder-Satchel
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Squad/prep.yml:163]: I think you meant to put the entity ID here?
```suggestion
      - id: RMCSatchelWelder
```

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]
Don't forget RMC for new IDs

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Squad/prep.yml (hunk 3 above)
Comment (Tunguso4ka) [reply Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Squad/prep.yml:163]: oh god.

**Review by DrSmugleaf — APPROVED** [review]
Thanks for the PR!

**Pull request merged**